src/Flights.groovy

```diff
@@ -0,0 +1,108 @@
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+class Flights {
+    static void main(String[] args) {
+        def flights = []
+        createFlights(flights)
+        def passengersToSof = flights.each { it.destination == "SOF"}.collect {it.passengers}.sum()
+        println("Passengers to Sofia - ${passengersToSof}")
+
+        def cap = flights.every {(double)it.passengers / (double)it.capacity >= 0.5}
+        println("Is every flight full at least 50% of its capacity? ${cap ? "Yes" : "No"} ")
+
+        // Is there a flight to London that is running late
+        def londonLate = flights.findAll {it.destination == "LON" }.any { it.runningLate}
+        println("Is there a flight to London running late? ${londonLate ? "Yes" : "No"}")
```

Comment:
Good one. 
Let's display which one(s) is running late

booleans should be named according to the Java convention "**isAnyLondonFlightLate**" for example